app/services/script-tags.ts

```diff
@@ -0,0 +1,118 @@
+/* eslint-disable no-mixed-spaces-and-tabs*/
```

Comment:
I had originally agreed. However, the format in which our Ember code creates the meta head tags made it problematic for this application. I have configured it so that only one script tag is created at a time with the properties appended in the same tag, not just content and name variables for a list array of metadata overrides. The latter creates multiple tags, with no inner content. There was a possibility to create a ScriptTagAttrs interface that included those properties within the meta-tags service, but there had still been an issue with how the tags were processed and created. If you like, we can review and make sure this is the best direction. There are, regardless, other methods that warranted the class. We can include this discussion with the aforementioned conversation about the Promise resolution as well.

Reply:
I am not sure I quite follow. Our Ember code does not deal with creating the head tags. We simply construct parameters and pass them to the `head-tags` service (which is implemented by an addon `ember-cli-head-tags`) to generate the tags for us. The `meta-tags` service is simply a wrapper over the `head-tags` service and is responsible for constructing the parameters for generating `<meta>` tags through the `head-tags` service. A service was needed because the there are quite a bit of data transformation to be down to construct these parameters.

But in this case, constructing the parameters for `<script>` tags is easy and straightforward, and could be done with maybe 2~3 lines of code.

Reply:
Again, I beleve your questions would benefit from a pair programming session. Let's schedule something and we can discuss further.